docs/developer-guide/ksqldb-rest-api/query-endpoint.md

```diff
@@ -37,7 +37,7 @@ Response JSON Object:
 
 - **header** (object): Information about the result.
     - **header.queryId**: (string): the unique id of the query. This can be useful when debugging. 
-    For example, when looking in the logs or processing log for errors or issues.
+    For example, when looking in the logs or processing logs for errors or issues.

```

Comment:
In this case _processing log_ refers to the singular `KSQL_PROCESSING_LOG`. I suggest leaving as-is or updating the change to use the explicit `KSQL_PROCESSING_LOG` term.

Reply:
It'd be good to specify `or the ksql processing log ...` to make it more explicit.